### PR TITLE
Expose data steward field in datamap response

### PR DESCRIPTION
**Pull request by TheAndrewJackson**
Closes fidesplus#721

### Code Changes

* [ ] Update the datamap export so include system managers
* [ ] Add test for change

### Steps to Confirm

* [ ] Run `nox -s "fides_env(test)"`
* [ ] Create a system
* [ ] Assigned a user to it
* [ ] Call `http://0.0.0.0:8080/api/v1/datamap/default_organization` and make sure the `system.users` key is returned and contains the system managers username

### Pre-Merge Checklist

* [ ] All CI Pipelines Succeeded
* Documentation:
  * [ ] documentation complete, [PR opened in fidesdocs](https://github.com/ethyca/fidesdocs/pulls)
  * [ ] documentation [issue created in fidesdocs](https://github.com/ethyca/fidesdocs/issues/new/choose)
* [ ] Issue Requirements are Met
* [ ] Relevant Follow-Up Issues Created
* [ ] Update `CHANGELOG.md`
* [ ] For API changes, the [Postman collection](https://github.com/ethyca/fides/blob/main/docs/fides/docs/development/postman/Fides.postman_collection.json) has been updated

### Description Of Changes

@adamsachs This is a slightly polished of what you saw when we paired. It contains a test that verifies the field in the response works as expected.

### CODE REVIEW

File: src/fides/api/ctl/sql_models.py [hunk 1]
```diff
@@ -287,7 +287,10 @@ class System(Base, FidesBase):
     ingress = Column(JSON)
 
     users = relationship(
-        "FidesUser", secondary="systemmanager", back_populates="systems"
+        "FidesUser",
+        secondary="systemmanager",
+        back_populates="systems",
+        lazy="selectin",
```
Comment (TheAndrewJackson) [inline src/fides/api/ctl/sql_models.py:293]: I ran into issues with this line in `export.py` 
```python
system_users = ", ".join([user.username for user in system.get("users", [])])
```

I needed to add this line to make resolving the linked users work in an async route






https://docs.sqlalchemy.org/en/14/orm/loading_relationships.html#select-in-loading

File: src/fides/api/ops/api/v1/endpoints/masking_endpoints.py [hunk 2]
```diff
@@ -4,6 +4,7 @@
 from loguru import logger
 from starlette.status import HTTP_400_BAD_REQUEST, HTTP_404_NOT_FOUND
 
+from fides.api.ops.api.v1.scope_registry import MASKING_EXEC, MASKING_READ
```
Comment (TheAndrewJackson) [inline src/fides/api/ops/api/v1/endpoints/masking_endpoints.py:7]: This was caught by isort/black 🤷‍♂️

File: src/fides/core/export.py [hunk 3]
```diff
@@ -148,6 +148,21 @@ def generate_system_records(  # pylint: disable=too-many-nested-blocks, too-many
     formatted_data_subjects, custom_data_subjects = format_data_subjects(
         server_resources["data_subject"]
     )
+
+    privacy_declaration_fields = [
```
Comment (TheAndrewJackson) [inline src/fides/core/export.py:152]: There used to be a magic variable below that was the number 12. This converts it to a be a dynamic variable

**Review by adamsachs — COMMENTED** [review]
really nice work @TheAndrewJackson! great job getting to the bottom of this relatively quickly, as well as making some incremental improvements to the API.

i think this is just about ready to go, i have a few nits that you can feel free to ignore if you don't find them valuable or if you don't have the time to address em. the only piece i'd like to see is the additional test coverage i mentioned - let me know if i can lend a hand with that.

oh also, looks like a full CI run hasn't executed against the latest here, i think because of the changelog conflict with `main`. so let's just get that fixed up quickly so we can be sure CI is all green!

File: src/fides/api/ctl/migrations/versions/39a23bc016ea_record_consent_preferences.py [hunk 4]
```diff
@@ -5,8 +5,8 @@
 Create Date: 2023-03-28 02:42:13.114938
 
 """
-from alembic import op
 import sqlalchemy as sa
+from alembic import op
```
[the remark this answers is not in the document]
Comment (TheAndrewJackson) [reply src/fides/api/ctl/migrations/versions/39a23bc016ea_record_consent_preferences.py:9]: It's not needed. I think it happened when I ran isort or black locally. I'm not sure why previous runs of those tools didn't catch this

File: src/fides/core/export.py [hunk 5]
```diff
@@ -239,6 +246,7 @@ def generate_system_records(  # pylint: disable=too-many-nested-blocks, too-many
                     system_custom_field_data[key_string] = ", ".join(value)
                 else:
                     system_custom_field_data[key_string] = value
+        system_users = ", ".join([user.username for user in system.get("users", [])])
```
[the remark this answers is not in the document]
Comment (TheAndrewJackson) [reply src/fides/core/export.py:249]: all good! the change has been made 🙂

File: src/fides/core/export.py [hunk 6]
```diff
@@ -300,6 +308,7 @@ def generate_system_records(  # pylint: disable=too-many-nested-blocks, too-many
                         system_dependencies,
                         system_ingress,
                         system_egress,
+                        system_users,
```
[the remark this answers is not in the document]
Comment (TheAndrewJackson) [reply src/fides/core/export.py:311]: I would but I think there are actually non PD related fields below 😭 

```
            "system.users",
            *privacy_declaration_fields, # non PD fields VVVVV
            "system.data_protection_impact_assessment.is_required",
            "system.data_protection_impact_assessment.progress",
            "system.data_protection_impact_assessment.link",
            "dataset.fides_key",
```

File: tests/ctl/api/test_datamap.py [hunk 7]
```diff
@@ -1037,3 +1096,17 @@ def test_datamap_two_custom_fields_one_multival_two_systems(
     assert (
         response.json() == EXPECTED_RESPONSE_TWO_CUSTOM_FIELDS_ONE_MULTIVAL_TWO_SYSTEMS
     )
+
+
+@pytest.mark.integration
+@pytest.mark.usefixtures("system_with_manager")
+def test_datamap_with_system_manager(
```
[the remark this answers is not in the document]
Comment (TheAndrewJackson) [reply tests/ctl/api/test_datamap.py:1103]: It shouldn't be too bad. I'll take a look now 🙂

File: src/fides/core/export.py (hunk 6 above)
[the remark this answers is not in the document]
Comment (adamsachs) [reply src/fides/core/export.py:311]: yuppp, good call. thanks for being diligent! nevermind then, we'll just have to tackle this in a bigger refactor

**Comment by TheAndrewJackson** [comment]
@adamsachs Everything looks good now. I think the one canceled job is related to the ops unit timeout

File: tests/ctl/api/test_datamap.py (hunk 7 above)
[the remark this answers is not in the document]
Comment (adamsachs) [reply tests/ctl/api/test_datamap.py:1138]: nice, thanks for adding this!

**Review by adamsachs — APPROVED** [review]
all looks good to me now, thanks for addressing the feedback!

**Pull request merged**